backend/src/middleware/notifications/notificationsMiddleware.ts

```diff
@@ -358,9 +359,12 @@ const notifyUsersOfMention = async (label, id, idsOfUsers, reason, context) => {
       mentionedCypher = `
       MATCH (postAuthor: User)-[:WROTE]->(post: Post)<-[:COMMENTS]-(comment: Comment { id: $id })<-[:WROTE]-(commenter: User)
       MATCH (user: User)
-      WHERE user.id in $idsOfUsers
-      AND NOT (user)-[:BLOCKED]-(commenter)
-      AND NOT (user)-[:BLOCKED]-(postAuthor)
+        WHERE user.id in $idsOfUsers
+        AND NOT (user)-[:BLOCKED]-(commenter)
```

Comment:
we also have to include muted for Notifications. Muted is included in Blocked. Muted silences all notifications.

Reply:
So then, what is the difference between muted and blocked?

Reply:
blocked should also hide all content of an user, muted should only hide notifications. But blocked also mutes the notifications.

Don't expect this to work properly, but we should ensure correct behaviour on notifications.

Reply:
I would like to do this in another PR, as testing is a lot of code and this issue gets out of scope